test/suites/dev-tanssi/registrar/test_registrar_deregister.ts

```diff
@@ -107,8 +107,8 @@ describeSuite({
 
                 // Check the number of keys in storage
                 const palletKeys = await polkadotJs.rpc.state.getKeys("0x3fba98689ebed1138735e0e7a5a790ab");
-                // 3 keys: version, registeredParas, pendingParas
-                expect(palletKeys.length).to.be.eq(3);
+                // 4 keys: version, registeredParas, pendingParas, pendingToRemove
+                expect(palletKeys.length).to.be.eq(4);
```

Comment:
I would add here a check that authorNoting gets cleaned, as well as credits. This should be easily doable by setting both storage items with a extrinsic